tools/android/MASTG-TOOL-0018.md

```diff
@@ -5,3 +5,21 @@ source: https://github.com/skylot/jadx
 ---
 
 jadx (Dex to Java Decompiler) is a command line and [GUI tool](https://github.com/skylot/jadx/wiki/jadx-gui-features-overview "jadx gui features overview") for producing Java source code from Android DEX and APK files - <https://github.com/skylot/jadx>
+
+## Decompiling via CLI
+
+The @MASTG-APP-0003 app can be decompiled using `jadx` by specifying the output directory and the target APK:
+
+```console
+jadx -d decompiled UnCrackable-Level1.apk
```

Comment:
Add the single class option since we use it as well in https://github.com/cpholguera/MASTestApp-Android/blob/main/tools/extract-code-for-mastg-demo.sh#L35